### DISCUSSION TITLE
How do I set a global .gitignore to work with Github Desktop on Mac?

### DISCUSSION THREAD

**Discussion by joshhubert-dsp** [community/community#33215] [Other Feature Feedback, Questions, & Ideas]
My goal is  to never have to manually ignore those pesky .DS_Store files again.
I don't see any options for a global config in the preferences, and couldn't find any advice for how to do it from the command line.

**Comment by airtower-luna** [comment]
See the [`gitignore` documentation](https://git-scm.com/docs/gitignore):

> Patterns which a user wants Git to ignore in all situations (e.g., backup or temporary files generated by the user’s editor of choice) generally go into a file specified by `core.excludesFile` in the user’s `~/.gitconfig`. Its default value is $XDG_CONFIG_HOME/git/ignore. If $XDG_CONFIG_HOME is either not set or empty, $HOME/.config/git/ignore is used instead.
